docs/database/elasticsearch.md

```diff
@@ -14,12 +14,8 @@ described on this page.
 
 ## Span Name
 
-The **span name** SHOULD be of the format `<endpoint id>`.
-
-The elasticsearch endpoint identifier is used instead of the url path in order to reduce the cardinality of the span
-name, as the path could contain dynamic values. The endpoint id is the `name` field in the
+The **span name** follows the [general database span name guidelines](database-spans.md#name) with the endpoint identifier stored in `db.operation.name`, instead of the url path in order to reduce the cardinality of the span and the index stored in `db.collection.name`. The endpoint id is the `name` field in the
```

Comment:
> instead of the url path in order to reduce the cardinality of the span

Is this part still relevant? why would users expect url path here? Maybe there was some prior art, but we don't really have to mention it here.

> The endpoint id is the `name` field in the
[elasticsearch schema](https://raw.githubusercontent.com/elastic/elasticsearch-specification/main/output/schema/schema.json).

since we're reusing `db.operation.name` attribute as is in the span name, consider moving this to the `db.operation.name` note and removing it from the span name section.

Reply:
Updated. 

> Is this part still relevant? why would users expect url path here?

That's a good point. Naturally people (and by that I mostly mean people who work on Elasticsearch and created the first instrumentations) pick the url path. But probably less relevant now. Removed it.


Also moved the link with the endpoint id into a note.